frontend/src/pages/dashboard/components/sidebar/components/end-shift-button.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button } from '~/libs/components/components.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import {
+  useAppDispatch,
+  useAppSelector,
+  useCallback,
+} from '~/libs/hooks/hooks.js';
+import { actions as driverActions } from '~/slices/driver/driver.js';
+import { ShiftStatus } from '~/slices/driver/libs/enums/shift-status.enum';
```

Comment:
It's missing .js and it's better to do a reexport.
```suggestion
import { ShiftStatus } from '~/slices/driver/libs/enums/enums.js';
```